lib/controller.js

```diff
@@ -607,7 +651,9 @@ const _PIPELINE_FUNCS = {
     parse_json: parse_json,
     pluck_path: json_pluck,
     equal_to: equal_to,
-    is_subset: is_subset
+    is_subset: is_subset,
+    in_superstring: in_superstring,
```

Comment:
Oops - I was testing a few different approaches and looks like I forgot to remove this one! Removed.